content/blog/understanding-the-dom/index.md

```diff
@@ -0,0 +1,519 @@
+---
+{
+	title: 'Understanding The DOM: How Browsers Show Content On-Screen',
+	description: 'Learn how the browser internally handles HTML and CSS to show the user webpages on-screen',
+	published: '2019-11-26T22:12:03.284Z',
+	authors: ['crutchcorn'],
+	tags: ['dom', 'browser internals'],
+	attached: [],
+	license: 'cc-by-nc-sa-4'
+}
+---
+
+Any web application relies on some fundamental technologies: HTML, CSS, and JavaScript. Even advanced front-end JavaScript frameworks such as Angular, React, or Vue will utilize some level of HTML to load the JavaScript. That said, how the browser handles HTML and CSS under-the-hood can be quite the mystery. In this article, I'm going to explain what the browser does to understand what it should show to the user.
+
+# The DOM {#the-dom}
+
+Just as the source code of JavaScript programs are broken down to abstractions that are more easily understood by the computer, so too is HTML. HTML, initially being derived from [SGML (the basis for XML as well)](https://en.wikipedia.org/wiki/Standard_Generalized_Markup_Language), actually _forms a tree structure in memory_ in order to [describe the relationships, layout, and executable tasks for items in the tree](#how-the-browser-uses-the-dom). This tree structure in memory is _called the Document Object Model_ (or _DOM_ for short).
```

Comment:
```suggestion
Just as the source code of JavaScript programs are broken down to abstractions that are more easily understood by the computer, so too is HTML. HTML, initially being derived from [SGML (the basis for XML as well)](https://en.wikipedia.org/wiki/Standard_Generalized_Markup_Language), _forms a tree structure in memory_ in order to [describe the relationships, layout, and executable tasks for items in the tree](#how-the-browser-uses-the-dom). This tree structure in memory is _called the Document Object Model_ (or _DOM_ for short).
```

Reply:
Again, this is just a tonal change by removing a word. Closing as a result